Stratis.Bitcoin.Features.BlockStore/LoopSteps/ProcessPendingStorageStep.cs

```diff
@@ -161,24 +156,25 @@ private StepResult PrepareNextBlockFromPendingStorage(ProcessPendingStorageConte
         /// <param name="context"><see cref="ProcessPendingStorageContext"/></param>
         private async Task PushBlocksToRepository(ProcessPendingStorageContext context)
         {
-            await this.BlockStoreLoop.BlockRepository.PutAsync(context.PendingBlockPairsToStore.First().ChainedBlock.HashBlock, context.PendingBlockPairsToStore.Select(b => b.Block).ToList());
+            this.logger.LogDebug(context.ToString());
 
+            await this.BlockStoreLoop.BlockRepository.PutAsync(context.PendingBlockPairsToStore.First().ChainedBlock.HashBlock, context.PendingBlockPairsToStore.Select(b => b.Block).ToList());
```

Comment:
Using `context.PendingBlockPairsToStore.First()` is not so intuitive right?
we could use  `context.PendingBlockPairToStore` instead.
But maybe a good idea is to assert that?

Reply:
Yes! I like it, will amend 👍